### DISCUSSION TITLE
hiding marketplace in eclipse framework

### DISCUSSION THREAD

**Discussion by marzikordi** [dbeaver/dbeaver#19408] [Q&A]
### Description

hi guys...
i have installed epsilon on my eclipse application but after that marketplace option is disappeared.
Eclipse Modeling Tools
Version: 2020-06 (4.16.0)

Meantime, when i want to install it again I encounter an error which showed that Marketplace is already installed.
So I dont know how I can get it back if it is hidden
I look forward to hearing from you and guide me 

thankd

### DBeaver Version

Eclipse Modeling Tools -Version: 2020-06 (4.16.0)

### Operating System

MacOs

**Comment by E1izabeth** [comment]
Could you explain, how is it connected with DBeaver?

**Comment by marzikordi** [comment]
there is no connected with DBeaver

**Comment by E1izabeth** [comment]
I'm not sure we can help you somehow
